santoku/utils/test/test_url_handler.py

```diff
@@ -1,209 +1,50 @@
-import os
-import sys
-
 import numpy as np
 import pandas as pd
 import pytest
 from santoku.utils.url_handler import InvalidURLError, URLHandler
 
 
-def get_raw_urls_num_subdomains_and_partial_domain():
+@pytest.mark.parametrize(
+    argnames=(
+        "input_url",
+        "num_subdomains",
+        "reference_partial_domain_raising_exception_for_invalid_url",
```

Comment:
There's enough context, no need to repeat the entire name of the function.
```suggestion
        "expected_partial_domain",
```

Reply:
Alternatively, since the result is not always a string we could simplify further
```suggestion
        "expected_result",
```